textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriter.java

```diff
@@ -0,0 +1,68 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.*;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.field.DataTuple;
+import edu.uci.ics.textdb.common.field.StringListField;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ */
+public class QueryRewriter implements IOperator{
```

Comment:
The name of `QueryRewriter` sounds too big. It actually a fuzzy query rewriter, right? Maybe we can choose a more concrete name for it.